src/Elastic.Apm/Model/Span.cs

```diff
@@ -47,14 +48,21 @@ internal class Span : ISpan
 			_enclosingTransaction = enclosingTransaction;
 			Name = name;
 			Type = type;
-			IsSampled = isSampled;
 
 			ParentId = parentId;
 			TraceId = traceId;
 
 			if (IsSampled)
-				// Started spans should be counted only for sampled transactions
-				enclosingTransaction.SpanCount.IncrementStarted();
+			{
+				// Started and dropped spans should be counted only for sampled transactions
+				if (enclosingTransaction.SpanCount.IncrementTotal() >= _configurationReader.TransactionMaxSpans)
```

Comment:
I think it should be strictly greater i.e.,
`if (enclosingTransaction.SpanCount.IncrementTotal() > _configurationReader.TransactionMaxSpans)`

Reply:
Changed